adb_graphics/default_specs.yml

```diff
@@ -1340,11 +1340,11 @@ vvort: # Vertical vorticity
     colors: vort_colors
     ncl_name: RELV_P8_2L103_{grid}_max1h
     ticks: 0
-    title: 0-1km Max Vertical Velocity (over prev hour)
+    title: 0-1km Max Vertical Vorticity (over prev hour)
     unit: 1/s
   mx02: # Hourly maximum of vertical vorticity over 0-2 km layer
     <<: *vvort 
-    title: 0-2km Max Vertical Velocity (over prev hour)
+    title: 0-2km Max Vertical Vorticity (over prev hour)
```

Comment:
I can't believe that we haven't caught this error yet.

Reply:
Wow! Nice catch!